package.json

```diff
@@ -17,68 +17,62 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "chalk": "^4.1.2",
-    "lit-element": "^2.5.1"
+    "chalk": "^5.2.0",
+    "lit": "^2.7.5"
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.3.0",
-    "@alaskaairux/icons": "^4.7.0",
-    "@webcomponents/webcomponentsjs": "^2.5.0",
-    "focus-visible": "^5.2.0"
+    "@aurodesignsystem/auro-icon": "^4.0.2"
   },
   "devDependencies": {
-    "@alaskaairux/auro-icon": "^3.4.0",
-    "@alaskaairux/design-tokens": "^3.12.2",
-    "@alaskaairux/icons": "^4.21.2",
+    "@alaskaairux/design-tokens": "^3.13.0",
+    "@alaskaairux/icons": "^4.27.0",
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.3.0",
-    "@babel/core": "^7.21.0",
+    "@aurodesignsystem/webcorestylesheets": "^4.5.0",
+    "@babel/core": "^7.21.8",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.21.0",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/runtime": "^7.21.0",
-    "@commitlint/cli": "^17.4.4",
-    "@commitlint/config-conventional": "^17.4.4",
-    "@open-wc/testing": "^3.1.7",
-    "@rollup/plugin-node-resolve": "^15.0.1",
-    "@semantic-release/changelog": "^6.0.2",
+    "@babel/plugin-transform-runtime": "^7.21.4",
+    "@babel/preset-env": "^7.21.5",
+    "@babel/runtime": "^7.21.5",
+    "@commitlint/cli": "^17.6.3",
+    "@commitlint/config-conventional": "^17.6.3",
+    "@open-wc/testing": "^3.1.8",
+    "@rollup/plugin-node-resolve": "^15.0.2",
+    "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@semantic-release/npm": "^9.0.2",
-    "@web/dev-server": "^0.1.35",
-    "@web/test-runner": "^0.15.0",
-    "@webcomponents/webcomponentsjs": "^2.7.0",
-    "autoprefixer": "^10.4.13",
+    "@semantic-release/npm": "^10.0.3",
+    "@types/node": "^20.2.4",
```

Comment:
What is this?

Reply:
not sure, I think it was to solve a problem in typescript installation.
But I tried to remove it and it still can build properly.